.env.example

```diff
@@ -1,46 +1,84 @@
+################################################
+# Brightpack Settings - used in local development only
+################################################
+
+APP_HOST="yourapp.test"
+APP_URL="http://${APP_HOST}"
+
+################################################
+# Craft General Settings (environments are 'dev', 'staging', or 'production')
+################################################
+
 ENVIRONMENT="dev"
 SECURITY_KEY=""
 APP_ID=""
 
-APP_HOST="yourapp.test"
-APP_URL="http://${APP_HOST}"
-PRIMARY_SITE_URL="${APP_URL}"
-PRIMARY_SITE_NAME="MS Craft"
+################################################
+# Database Settings
+################################################
 
 DB_DRIVER="mysql"
-DB_SERVER="localhost"
+DB_SERVER="127.0.0.1"
 DB_USER="root"
 DB_PASSWORD=""
 DB_DATABASE=""
 DB_SCHEMA="public"
 DB_TABLE_PREFIX=""
 DB_PORT=""
 
+################################################
+# Primary Site Settings (the only site in a single site install)
+################################################
+
+PRIMARY_SITE_URL="http://${APP_HOST}"
+PRIMARY_SITE_NAME="MS Craft"
+
+################################################
+# AWS Hosted Asset Volumes and CDN
+################################################
+
 AWS_DEFAULT_REGION="us-east-1"
 AWS_S3_ACCESS_KEY_ID=""
 AWS_S3_SECRET_ACCESS_KEY=""
 AWS_S3_BUCKET=""
 
-CLOUDFRONT_URL=""
-CLOUDFRONT_DISTRIBUTION_ID=""
+# CLOUDFRONT_URL=""
+# CLOUDFRONT_DISTRIBUTION_ID=""
 
-IMGIX_HOST=""
-IMGIX_URL="https://${IMGIX_HOST}"
-IMGIX_API_KEY=""
+################################################
+# IMGIX Settings (if used)
+################################################
+
+# IMGIX_HOST=""
+# IMGIX_URL="https://${IMGIX_HOST}"
+# IMGIX_API_KEY=""
+
+################################################
+# System Email Settings
+################################################
 
 SYSTEM_EMAIL_NAME="MS Craft"
 SYSTEM_EMAIL_ADDRESS=""
+SYSTEM_EMAIL_REPLY_TO_ADDRESS=""
 SENDGRID_API_KEY=""
 
-STRIPE_PUBLIC=""
-STRIPE_SECRET=""
+################################################
+# Other Integrations (if required) 
+################################################
+
+# STRIPE_PUBLIC=""
+# STRIPE_SECRET=""
+
+# MAILCHIMP_API_KEY=""
+# MAILCHIMP_AUDIENCE_ID=""
 
-MAILCHIMP_API_KEY=""
-MAILCHIMP_AUDIENCE_ID=""
+# RECAPTCHA_SITE_KEY=""
+# RECAPTCHA_SECRET_KEY=""
 
-RECAPTCHA_SITE_KEY=""
-RECAPTCHA_SECRET_KEY=""
+################################################
+# Mostly Serious QA and Support
+################################################
 
-MARKERIO_DESTINATION=""
+# MARKERIO_DESTINATION=""
+# SENTRY_DSN=""
```

Comment:
@mikeatmostlys We use Marker and Sentry by default on every project so these should probably not be commented it out. Seems like you're commenting out things that are optional, but these are standard.

Reply:
Good to know will remove comments.